wasm/src/main/java/com/dylibso/chicory/wasm/types/ValueType.java

```diff
@@ -118,6 +118,15 @@ public boolean isReference() {
         }
     }
 
+    public boolean isVec() {
+        switch (this) {
+            case V128:
+                return true;
+            default:
+                return false;
+        }
```

Comment:
Do we expect more types in the future? I only see `v128` in the specification.
```suggestion
        return this == V128;
```
Also, the method is currently unused. If there is only one vector type, seems better to just compare against the enum value.

Reply:
removed